releases/v21.1.0.md

```diff
@@ -0,0 +1,142 @@
+Core | **Inverted Joins** | CockroachDB now supports [inverted joins](../v21.1/joins.html#inverted-joins), which force the optimizer to use an inverted index on the right side of the join. Inverted joins can only be used with `INNER` and `LEFT` joins.
+Core | **Partial Inverted Indexes** | You can now create a [partial inverted index](../v21.1//inverted-indexes.html#partial-inverted-indexes) on a subset of `JSON`, `ARRAY`, or geospatial container column data.
+Core | **Virtual Computed Columns** | You can now create [virtual computed column](../v21.1/computed-columns.html), which are not stored on disk and are recomputed as the column data in the expression changes.
+Core | **Renaming and Dropping User-Defined Types** | It's now possible to [rename a user-defined type](../v21.1/alter-type.html#rename-a-user-defined-type) and [drop a value in a user-defined type](../v21.1/alter-type.html#drop-a-value-in-a-user-defined-type).
+Core | **Sequence Caching** | You can now create a sequence with the [`CACHE`](../v21.1/create-sequence.html#cache-sequence-values-in-memory) keyword to have the sequence cache its values in memory.  
+Core | **Changing Sequence & View Ownership** | You can use the new `OWNER TO` parameter to change the owner of a [sequence](../v21.1/alter-sequence.html) or [view](../v21.1/alter-view.html).  
+Core | **Show `CREATE` Statements for the Current Database** | You can now use [`SHOW CREATE ALL TABLES`](../v21.1/show-create.html#show-the-statements-needed-to-recreate-all-tables-views-and-sequences-in-the-current-database) to return the `CREATE` statements for all of the tables, views, and sequences in the current database.
+Core | **ORM Compatibility for Spatial** | [Spatial libraries for Hibernate, ActiveRecord, and Django](../v21.1/spatial-data.html#orm-compatibility) are now fully compatible with CockroachDB's spatial features.
```

Comment:
To add: New empty, unimplemented `pg_catalog` tables see https://www.cockroachlabs.com/docs/v21.1/pg-catalog.html